pins/haskell-nix.json

```diff
@@ -1,7 +1,7 @@
 {
   "url": "https://github.com/input-output-hk/haskell.nix",
-  "rev": "3584345a9ab001d1867e972a1a20b4406cbffd68",
-  "date": "2019-02-14T09:35:57+08:00",
-  "sha256": "08pzfvspfl5nvn5szy7bv3rbwymjgmbdm1ac571c64fzhrwf5ghw",
+  "rev": "c6106e3ac07a9f782791f9de750c15a5f13718a1",
```

Comment:
- [ ] replace with actual commit after merge of https://github.com/input-output-hk/haskell.nix/pull/76